mangaki/mangaki/utils/anidb.py

```diff
@@ -183,6 +181,33 @@ def get_or_update_work(self,
             if title_type == 'synonym':
                 synonyms[lang] = title
 
+        # Handling of staff
+        creators = []
+        staff_map = dict(Role.objects.filter(slug__in=['author', 'director', 'composer']).values_list('slug', 'pk'))
+        for creator_node in all_creators.find_all('name'):
+            creator = str(creator_node.string).strip()
+            creator_id = creator_node.get('id')
+            creator_type = creator_node.get('type')
+            staff_id = None
+
+            if creator_type == 'Direction':
+                staff_id = 'director'
+            elif creator_type == 'Music':
+                staff_id = 'composer'
+            elif creator_type == 'Original Work' or creator_type == 'Story Composition':
+                staff_id = 'author'
+            elif creator_type == 'Animation Work' or creator_type == 'Work':
+                # AniDB marks Studio as such a creator's type
+                studio, s_created = Studio.objects.get_or_create(title=creator)
```

Comment:
We should definitely add a new field called `anidb_aid` to `Studio` I guess.
Let's do this in another PR.